src/patternfly/components/Select/docs/select-checkbox.md

```diff
@@ -0,0 +1,32 @@
+The checkbox select can select multiple items using checkboxes. The number of items selected is reflected in a badge in the dropdown toggle. The user may clear items by unchecking or using the clear button. Optionally, items may be grouped.
+
+## Accessibility
+
+*This section to be updated once the React implementation is complete.*
+
+| Attribute | Applied To | Outcome |
+| -- | -- | -- |
+| `role` or `aria` | `pf-c-select` |  accessibility notes. |
+*Note:* The attribute `aria-pressed="true"` should be set programmatically to the active item.
+
+
+## Usage
+
+| Class | Applied To | Outcome |
+| -- | -- | -- |
+| `.pf-c-select` | `<div>` |  Initiates a custom select. |
+| `.pf-m-expanded` | `.pf-c-select` |  Indicates the select is expanded. |
+| `.pf-c-select__toggle` | `<div>` |  Initiates a custom toggle. |
+| `.pf-m-typeahead` | `.pf-c-select__toggle` |  Indicates the select has a typeahead. |
+| `.pf-c-select-toggle__wrapper` | `<div>` |  Initiates a custom select toggle wrapper so that chips and input field can wrap together. |
+| `.pf-c-pf-c-chip` | `<div>` |  Initiates a chip. (See chip component for more details) |
+| `.pf-c-select-toggle__typeahead` | `<input class="pf-c-form-control">` |  Initiates the input field for typeahead. |
```

Comment:
what about using `input.pf-c-form-control` here? Similar to the a11y section in the [button component](https://pf-next.com/components/Button/examples/) where we use `button.pf-c-button` and `a.pf-c-button`

Reply:
Also if you do that, it applies to a couple more elements in this table